integration-cli/docker_cli_build_test.go

```diff
@@ -2293,6 +2293,29 @@ func TestBuildExposeOrder(t *testing.T) {
 	logDone("build - expose order")
 }
 
+func TestBuildExposeHostPort(t *testing.T) {
```

Comment:
@duglin 
You mean we should fail when test EXPOSE ip:hostPort:containerPort?

Also, what does "then verified the CLI saw it" mean,
could you please give me some hints?

Thanks.